components/tikv_kv/src/lib.rs

```diff
@@ -119,12 +120,25 @@ impl WriteData {
             modifies,
             extra,
             deadline: None,
+            disk_full_opt: DiskFullOpt::NotAllowedOnFull,
         }
     }
 
     pub fn from_modifies(modifies: Vec<Modify>) -> Self {
         Self::new(modifies, TxnExtra::default())
     }
+
+    pub fn set_allowed_on_almost_full(&mut self) {
```

Comment:
Just `set_disk_full_opt` is fine. This function name is bad, because it doesn't show **what** fulll. Should point out **disk** if you would like to keep it.

Reply:
Done